netbout-web/src/test/java/com/netbout/rest/TkAppTest.java

```diff
@@ -87,4 +93,80 @@ public void rendersStaticResources() throws Exception {
         }
     }
 
+    /**
+     * TkApp can redirect unauthenticated users to login page
+     * and store return location in RsReturn cookie.
+     * @todo #609:30min Currently there is no way to send unauthorized
+     *  request from a test, as there is a PsFake(TkAppAuth.TESTING)
+     *  in TkAppAuth, hence all requests are authorized.
```

Comment:
@komaz Can you reword this ticket? I can see a factual statement ("no way to send unauthorized request... hence all requests are authorized"), however, there's nothing that indicates exactly what needs to be fixed.